rio_stac/stac.py

```diff
@@ -1,18 +1,21 @@
 """Create STAC Item from a rasterio dataset."""
 
 import datetime
+import math
 import os
 import warnings
 from contextlib import ExitStack
 from typing import Any, Dict, List, Optional, Tuple, Union
 
+import numpy
 import pystac
 import rasterio
 from rasterio import warp
 from rasterio.io import DatasetReader, DatasetWriter, MemoryFile
 from rasterio.vrt import WarpedVRT
 
 PROJECTION_EXT_VERSION = "v1.0.0"
+RASTER_EXT_VERSION = "v1.0.0"
```

Comment:
will update to `1.1.0` once https://github.com/stac-extensions/raster/pull/20 is merged